app/src/main/java/app/foodfinderapp/dto/Review.kt

```diff
@@ -1,8 +1,9 @@
 package app.foodfinderapp.dto
 
 import android.media.Rating
+import com.google.gson.annotations.SerializedName
 
-data class Review(var reviewID: Int, var reviewTitle: String, var reviewDescription: String, var reviewRating: Float) {
+data class Review(var reviewID: Int, @SerializedName("reviewTitle") var reviewTitle: String,@SerializedName("reviewDescription") var reviewDescription: String, var reviewRating: Float) {
```

Comment:
See notes above.

@SerializedName is only required if the JSON field names do not match the data class attribute names.
In a case like this, where the names match, @SerializedName is not providing value... and thus, is extra code that is not needed.